scripts/qa_crowbarsetup.sh

```diff
@@ -1927,7 +1943,13 @@ function onadmin_addupdaterepo()
 {
     pre_hook $FUNCNAME
 
-    local UPR=/srv/tftpboot/repos/Cloud-PTF
+    onadmin_set_source_variables
+
+    if iscloudver 5plus; then
```

Comment:
we only need $cloudsource which we have available from the very start of the script
and we might want to do it after definition of iscloudver function, then line 2415 is good (after ruby= is set in a similar way)